infrastructure/variables.tf

```diff
@@ -79,6 +79,10 @@ variable "transformation_url_probate" {
   default = ""
 }
 
+variable "update_url_bulkscan" {
+  default = "http://bulk-scan-sample-app-aat.service.core-compute-aat.internal"
```

Comment:
there's no need for this variable - can just simply assign directly in main app settings

Reply:
it's consistent with transformation though

Reply:
yes. it is. and it has no value in it. we moved to aks, aks defined in all envs. this terraform change is legacy only applied in old demo environment. just keep in consistent and define in main.tf. if endpoint doesn't work it'll just be smth azure cloud provides ;)